CSS/components/badge.css

```diff
@@ -0,0 +1,49 @@
+.badge{
+    display: flex;  
+    position: relative;
+    cursor: pointer;
+    width: max-content;
+}
+.badge-icon{
+    width: 2rem;
+    height: 2rem;
+}
+.badge-value{
+    display: flex;
+    justify-content: center;
+    padding: 0.5rem;
+    align-items: center;
+    position: absolute;
+    background-color: var(--primary-color);
+    color: var(--background-color);
+    width: 1rem;
```

Comment:
You can increase badge width for icons. That 10 is not clearly visible. make badge width for at least 2 digit numbers fit in.

Reply:
Thanks for pointing out adjusted the size

Reply:
![Screenshot (414)](https://user-images.githubusercontent.com/75560307/152690403-de170a1f-6c54-48f7-8dc8-137a83c9dc78.png)